src/builder/openChrome.applescript

```diff
@@ -0,0 +1,49 @@
+(*
+Copyright (c) 2015-present, Facebook, Inc.
+All rights reserved.
+
+This source code is licensed under the BSD-style license found in the
+-- LICENSE file in the root directory of this source tree. An additional grant
+of patent rights can be found in the PATENTS file in the same directory.
+*)
```

Comment:
I saw probably 5 other implementations of the script and I'm not sure who was first to give attribution to.

At the end of the day, this code is just a system level utility so I'm not sure why you'd need copyrights on it.

Up to you what you want to do with it. All I know is that I want this to make my Dev flow faster :)